quantities/quantities/Energy.cs

```diff
@@ -1,82 +1,24 @@
 using System.Numerics;
 using Quantities.Dimensions;
+using Quantities.Factories;
 using Quantities.Measures;
 using Quantities.Prefixes;
 using Quantities.Units.Imperial;
+using Quantities.Units.NonStandard;
 using Quantities.Units.Si;
 
 namespace Quantities.Quantities;
 
 public readonly struct Energy : IQuantity<Energy>, IEnergy<Mass, Length, Time>
-    , ISiDerived<Energy, IEnergy>
-    , IImperial<Energy, IEnergy>
+    , IFactory<ICompoundFactory<Energy, IEnergy<Mass, Length, Time>>, Energy.Factory<LinearTo>, Energy.Factory<LinearCreate>>
     , IDivisionOperators<Energy, Time, Power>
     , IDivisionOperators<Energy, Power, Time>
 {
     private readonly Quant quant;
```

Comment:
You're quite right. The "root" concept is - as of now - more an Idea that I planted in some quantity types, rather than a full blown concept.
What I don't like is that it introduces a dependency to units. I'd rather the quantities didn't know about any concrete unit at all.

A design question we should try and solve together. See also issue #10.